src/account/operations/transfer/high_level/sweep_outputs.rs

```diff
@@ -0,0 +1,412 @@
+
+            Ok(transaction_ids)
+        }
+        .boxed()
+    }
+
+    pub(crate) async fn send_sweep_transaction(
+        &self,
+        address: AddressWrapper,
+        output_ids: impl IntoIterator<Item = OutputId>,
+        outputs: impl IntoIterator<Item = Output>,
+    ) -> crate::Result<TransactionId> {
+        let mut custom_inputs = Vec::new();
+        let mut custom_outputs = Vec::new();
+
+        match address.inner {
+            Address::Alias(alias_address) => {
+                let (output_id, alias_output) = self
+                    .output_id_and_next_alias_output_state(*alias_address.alias_id())
+                    .await?;
+                custom_inputs.push(output_id);
+                custom_outputs.push(Output::Alias(alias_output));
+            }
+            Address::Nft(nft_address) => {
+                let (output_id, nft_output) = self.output_id_and_nft_output(*nft_address.nft_id()).await?;
+                custom_inputs.push(output_id);
+                custom_outputs.push(Output::Nft(nft_output));
+            }
+            Address::Ed25519(_) => {
+                return Err(Error::BurningFailed(
+                    "Ed25519 address is not intended to be swept".to_string(),
+                ));
+            }
+        }
+
+        custom_inputs.append(&mut output_ids.into_iter().collect::<Vec<_>>());
+        custom_outputs.append(&mut outputs.into_iter().collect::<Vec<_>>());
+
+        let transfer_options = Some(TransferOptions {
+            custom_inputs: Some(custom_inputs),
+            try_fetch_unfound_outputs: true,
+            ..Default::default()
+        });
+
+        let transfer_result = self.send(custom_outputs, transfer_options).await?;
+        match &transfer_result.message_id {
+            Some(message_id) => {
+                let _ = self.client.retry_until_included(message_id, None, None).await?;
+                let _ = self.sync(None).await?;
+            }
+            None => return Err(Error::BurningFailed("Could not sweep address outputs".to_string())),
+        }
+
+        Ok(transfer_result.transaction_id)
+    }
+
+    async fn fetch_governor_address_alias_outputs(
+        &self,
+        address: &AddressWrapper,
+    ) -> crate::Result<Vec<OutputResponse>> {
+        let alias_query_parameters = vec![QueryParameter::Governor(address.to_bech32())];
+
+        let alias_output_ids = self.client.aliases_output_ids(alias_query_parameters).await?;
+        let output_responses = self.client.get_outputs(alias_output_ids).await?;
+
+        Ok(output_responses)
+    }
+
+    async fn fetch_address_basic_outputs(&self, address: &AddressWrapper) -> crate::Result<Vec<OutputResponse>> {
+        let query_parameters = vec![
+            QueryParameter::Address(address.to_bech32()),
+            QueryParameter::HasExpirationCondition(false),
+            QueryParameter::HasTimelockCondition(false),
+            QueryParameter::HasStorageDepositReturnCondition(false),
```

Comment:
I think because of this the comment for the high level method should also say, that such outputs could get ignored